pom.xml

```diff
@@ -72,8 +72,8 @@
 		<!-- Version 2.7.0 is the last free version. The latest libraries are shipped only with the commercial JGoodies Smart Client -->
 		<jgoodies-looks-version>2.7.0</jgoodies-looks-version>
 
-		<!-- Skip 4.65-1.05-RC because it doesn't work, author is working on fixing it -->
-		<sevenzip-version>4.65-1.04-RC</sevenzip-version>
+		<!-- Skip 4.65-1.05-RC because it doesn't work, author is working on fixing it -->		
+		<sevenzip-version>9.20-2.00beta</sevenzip-version><!--  Maybe ready to be removed, replaced with java-unrar -->
```

Comment:
@acanive it could be removed but all methods used from it must be replaced by methods from the `java-unrar`.

Reply:
> @acanive it could be removed but all methods used from it must be replaced by methods from the `java-unrar`.

I'm working on it